Leibit.Core/Data/sued/TF.xml

```diff
@@ -0,0 +1,412 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<estw>
+  <station name="Radolfzell" short="RRZ" refNr="56">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="66BP" direction="right" />
+      <block name="56BN2" direction="right" />
+    </track>
+  </station>
+
+  <station name="Schwackenreute" short="RSU" refNr="65">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="65BN" direction="left"/>
+    </track>
+  </station>
+
+  <station name="Eriskirch" short="TEK" refNr="76">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="77BP2" direction="left"/>
+    </track>
+  </station>
+
+  <station name="Baienfurt" short="TBAG" refNr="40">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="40BN2" direction="right"/>
+    </track>
+  </station>
+
+  <station name="Aulendorf" short="TAU" refNr="38">
+    <track isPlatform="false" calculateDelay="false">
+      <block name="38BP69" direction="left"/>
```

Comment:
Die ersten ZN-Felder aus dieser Richtung müssten "N2" und "66" sein.

Reply:
genau, beide aber auch wieder ohne eine Betriebsstelle vorangesetzt.

Reply:
Wie genau meinst du das? Vielleicht kurz zum Hintergrund... Diese Einträge sind dazu da, damit Züge, die aus dem Nachbarbereich kommen, eine sinnvolle aktuelle Betriebsstelle bekommen. Wenn man erst die Blöcke vor dem ESig nimmt, bleibt die akt. Betriebsstelle für eine ganze Zeit leer. Deswegen sollten immer die ersten ZN-Felder aus der jeweiligen Richtung genommen werden, wobei die Voranzeigen (V im Namen) nicht von ESTWsim übertragen werden.

Reply:
Ich meinte mit Betriebsstelle die Ziffern der Betriebsstellen, die meistens ja vor dem "N2" stehen. Konnte ich beide hier in der Sim wieder nicht ablesen.

Reply:
Achso. Dann am besten mal wie oben beschrieben versuchen. Also manuell eine ZN eintippen und dann in der .dat Datei nachschauen, wie der Block wirklich heißt.